### PR TITLE
Applying Traefik image pull from ACR by using imperative instructions in step 08

### PR DESCRIPTION
**Pull request by magrande**

### CODE REVIEW

**Review by ckittel — CHANGES_REQUESTED** [review]
Looks good to me.

However, I think this is the LAST usage of docker.io now right?  If so, shouldn't we have an Azure Policy change to go with this PR?  https://github.com/mspnp/aks-baseline-multi-region/blob/main/cluster-stamp.json#L1718  (while you're in there, you can fix the unescaped `.`s as well, that would be good)

**Comment by ckittel** [comment]
Also, we should be able to remove an _Azure Firewall_ rule now for docker.io.  That was listed in the task as being inscope for this work.

File: networking/hub-region.v1.1.json [hunk 1]
```diff
@@ -704,9 +704,7 @@
                                         ],
                                         "fqdnTags": [],
                                         "webCategories": [],
-                                        "targetFqdns": [
-                                            "docker.io"
-                                        ],
+                                        "targetFqdns": [],
```
Comment (ckittel) [inline networking/hub-region.v1.1.json:707]: You need to remove the whole rule :)  Not just leave it empty.

File: cluster-stamp.json [hunk 2]
```diff
@@ -1715,7 +1715,7 @@
                 "policyDefinitionId": "[variables('policyResourceIdEnforceImageSource')]",
                 "parameters": {
                     "allowedContainerImagesRegex": {
-                        "value": "[concat(variables('containerRegistryName'),'.azurecr.io/.+$|mcr.microsoft.com/.+$|docker.io/library/traefik.+$')]"
+                        "value": "[concat(variables('containerRegistryName'),'.azurecr.io/.+$|mcr.microsoft.com/.+$')]"
```
Comment (ckittel) [inline cluster-stamp.json:1718]: Can you fix the non-escaped control character problem that I mentioned as well?

**Review by ckittel — CHANGES_REQUESTED** [review]

File: networking/hub-region.v1.1.json (hunk 1 above)
Comment (magrande) [reply networking/hub-region.v1.1.json]: Sorry, I was looking at the leave (not even the tree.. ) instead of forest

Comment (ckittel) [reply networking/hub-region.v1.1.json]: Haha, love the analogy :)

File: cluster-stamp.json (hunk 2 above)
Comment (ckittel) [reply cluster-stamp.json:1718]: If you're looking for a reference for what I mean here check out: https://github.com/mspnp/aks-baseline-regulated/blob/main/cluster-stamp.json#L2381

Comment (ckittel) [reply cluster-stamp.json:1718]: Also, you'll want to update the description a few lines above this regex to match :)

Comment (magrande) [reply cluster-stamp.json:1718]: you read my mind thanks

Comment (ckittel) [reply cluster-stamp.json:1718]: tenor

Comment (ckittel) [reply cluster-stamp.json]: Nailed it!

**Review by ckittel — APPROVED** [review]
🚀

**Comment by magrande** [comment]
Thanks @ckittel !! I'm gonna test the hell out it before merging

**Pull request merged**